docs/tutorial.md

```diff
@@ -0,0 +1,290 @@
+```
+
+Review the contents of the `validation.yaml` file. The file contains the configuration for the validation. The `spec` section contains the configuration for the validation.  Change the `spec` section to match your requirements. For example, you can change the `defaultRegion` to match your prefered AWS region. 
+
+
+```yaml
+apiVersion: validation.spectrocloud.labs/v1alpha1
+kind: AwsValidator
+metadata:
+  name: awsvalidator-spectro-cloud-base
+spec:
+  auth: {}
+  defaultRegion: us-west-1
+  iamRoleRules:
+  - iamPolicies:
+    - name: Controllers Policy
+      statements:
+      - actions:
+      # Remainder of the file omitted for brevity
+```
+
+
+Now that you have the validation configuration, you can create the validation. Use the following command to create the validation.
+
+```shell
+kubectl apply --values validation.yaml
+```
+
+You can verify the Custom Resource (CR) for the AWS plugin was created by using the following command. 
+
+```shell
+kubectl get crd
+```
+
+```shell
+NAME                                             CREATED AT
+awsvalidators.validation.spectrocloud.labs       2023-11-14T23:47:03Z
+validationresults.validation.spectrocloud.labs   2023-11-14T23:46:54Z
+validatorconfigs.validation.spectrocloud.labs    2023-11-14T23:46:54Z
+```
+
+Review the result of the validation by using the following command.
+
+```shell
+kubectl describe validationresults
+```
+
+```shell
+Name:         validator-plugin-aws-awsvalidator-spectro-cloud-base
+Namespace:    default
+Labels:       <none>
+Annotations:  validator/validation-result-hash: 2IjYPs4C+8fu6MnuBq09lg==
+API Version:  validation.spectrocloud.labs/v1alpha1
+Kind:         ValidationResult
+Metadata:
+  Creation Timestamp:  2023-11-15T17:22:33Z
+  Generation:          1
+  Resource Version:    141250
+  UID:                 18d37dd9-9e4a-4397-b857-03116cf62975
+Spec:
+  Expected Results:  1
+  Plugin:            AWS
+Status:
+  Conditions:
+    Failures:
+      v1alpha1.IamRoleRule SpectroCloudRole missing action(s): [s3:PutObject s3:DeleteObject s3:PutBucketOwnershipControls s3:PutBucketPolicy s3:PutBucketPublicAccessBlock s3:PutObjectAcl] for resource arn:*:s3:::* from policy Controllers Policy
+    Last Validation Time:  2023-11-15T17:26:37Z
+    Message:               One or more required IAM permissions was not found, or a condition was not met
+    Status:                False
+    Validation Rule:       validation-SpectroCloudRole
+    Validation Type:       aws-iam-role-policy
+  Sink State:              N/A
+  State:                   Failed
+Events:                    <none>
+```
+
+If the validation fails, you can review the `Failures` section of the `Conditions` section to determine the cause of the failure. In this example, the IAM role is missing the required permissions. You can update the IAM role to add the missing permissions and rerun the validation.
```

Comment:
"of the `Conditions` section to" -> "of each `Condition` in its `Status` to"

Reply:
"and rerun the validation" -> "and wait 30s for the validation to be revalidated"